forch/forchestrator.py

```diff
@@ -37,10 +37,29 @@
 _BEHAVIORAL_CONFIG_DEFAULT = 'faucet.yaml'
 _SEGMENTS_VLAN_DEFAULT = 'segments-to-vlans.yaml'
 _DEFAULT_PORT = 9019
-_FAUCET_VARZ_HOST = '127.0.0.1'
-_FAUCET_PROM_PORT = 9302
-_GAUGE_VARZ_HOST = '127.0.0.1'
-_GAUGE_PROM_PORT = 9303
+_FAUCET_HOST_DEFAULT = '127.0.0.1'
```

Comment:
HOSTs aren't defaults...  They are only "default" if they can be changed with a config -- like with the ports -- but if it's always the value, then it's not a _DEFAULT.

Reply:
Removed DEFAULT